\documentclass[]{deedy-resume-openfont}
\begin{document}
\lastupdated

\namesection{}{Ricardo Avila}{
%Github:// \href{https://github.com/ravila4}{\custombold{github.com/ravila4}} \\
{ravila11@miners.utep.edu} | 915.781.0945
}

\begin{minipage}[t]{0.33\textwidth} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     EDUCATION
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Education} 

\subsection{U. of Texas at El Paso}
\descript{MS in Bioinformatics}
\location{In Progress | El Paso, TX}
\sectionsep

\descript{BS in Biochemistry}
\location{May 2016 | El Paso, TX}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     LINKS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Links} 
Github:// \href{https://github.com/ravila4}{\bf ravila4}\\
Linkedin:// \href{https://www.linkedin.com/in/ravila4/}{\bf ravila4}\\
Twitter://  \href{https://twitter.com/ravila_4}{\bf @ravila\_4}\\
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     COURSEWORK
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Coursework}
\subsection{Graduate}
Comp. Modeling of Biostructures\\
Statistical Analysis \\
Statistical Programming \\
Bioinformatics I \& II \\
Postgenomic Analysis\\
Data Mining \\
Machine Learning \\
Database Management\\
Systems Neuroscience\\
\sectionsep

\subsection{Undergraduate}
Biochemistry I \& II \\
Structural Biochemistry\\
Molecular Cell Biology \\
Prokaryotic Molecular Biology \\
Organic Chemistry I \& II \\
Physical Chemistry II \\
% Molecular Symmetry \& Group Theory \\
Discrete Mathematics \\
Data Structures and Algorithms\\
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     SKILLS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Skills}

\subsection{Laboratory}
 \textbullet{} protein expression
 \textbullet{} SDS-PAGE\\
 \textbullet{} UV-Vis
 \textbullet{} crystal screening
 \textbullet{} HPLC\\
% \textbullet{} PCR
% \textbullet{} plasmid purification
% \textbullet{} NMR \\
\sectionsep

\subsection{Programming}
 \textbullet{} Python
 \textbullet{} UNIX Shell
 %\textbullet{} Java
 \textbullet{} R \\
 \textbullet{} MySQL
\sectionsep

\subsection{Software \& Toolkits}
 %\textbullet{} Git
 %\textbullet{} Docker
 \textbullet{} Chimera
 \textbullet{} PyMol\\
 %\textbullet{} RDKit
 %\textbullet{} AutoDock
 %\textbullet{} OpenBabel
 %\textbullet{} Rosetta
 %\textbullet{} EMAN2
 %\textbullet{} Relion
 %\textbullet{} VMD
 \textbullet{} Schrödinger
 \textbullet{} CCG MOE
 \\
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     LANGUAGES
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Languages}
\location{Native fluency:}
 \textbullet{}   English \textbullet{} Spanish \\

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     COLUMN TWO
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\end{minipage} 
\hfill
\begin{minipage}[t]{0.66\textwidth} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Objective
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{About Me}
Biochemist with knowledge of structural biochemistry, bioinformatics, and a mixture of wet/dry lab experience. Interested in drug discovery research and protein design. %applying data mining and machine learninghttps://www.overleaf.com/project/5aaf38b1fcb2803f241ffe81 algorithms to problems in structural bioinformatics, drug discovery, and high dimensional -omics data analysis.
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Work Experience
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Work Experience}
\runsubsection{GlaxoSmithKline, via Atrium}
\descript{| Protein Design and Informatics Intern}
\location{May 2018 – August 2018 | Upper Providence, PA}
\sectionsep
\begin{tightemize}
\item Evaluated methods for identifying cryptic binding pockets in proteins.\item Wrote scripts to automate analysis of molecular dynamics (MD) data. \item Ran scalability benchmarks to optimize MD simulation performance.  \end{tightemize}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     RESEARCH
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Research}
\runsubsection{Structure-Based Drug Discovery Lab}
\descript{| Graduate RA}
\location{May 2016 – Present | UTEP School of Pharmacy | El Paso, TX}
\begin{tightemize}
\item Development and testing of novel machine-learning scoring functions for the prediction of drug-target binding affinities. \item Learned to utilize the high performance computing resources at the Texas Advanced Computing Center (TACC). \item Developed pipeline for in silico drug discovery targeting GPCRs.% \item Contributed to open source projects.
\end{tightemize}
\sectionsep

%\runsubsection{Structure-Based Drug Discovery}
%\descript{| Self-Guided Graduate Researcher}
%\location{May 2016 – December 2017 | UTEP Bioinformatics Program | El Paso, TX}
%\begin{tightemize}
%\item  Worked on putting together a workflow for virtual screening %drugs to GPCR targets.\item Learned to use several tools for %molecular docking and virtual screening, including AutoDock and %Schrodinger's drug discovery suite. \end{tightemize}
%\sectionsep

\runsubsection{Structural Biochemistry Lab}
\descript{| Undergraduate RA}
\location{March 2014 – August 2015 | UTEP Chemistry Department | El Paso, TX}
\begin{tightemize}
\item Transformation of recombinant plasmids in BL21 cells. \item Bacterial expression and purification of chaperonin proteins.\item Functional assays to evaluate protein folding activity. \item Cryo-EM image processing for 3D structure determination.\end{tightemize}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Workshops
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Workshops}
\runsubsection{MMTF Workshop \& Hackathon}\\
\location{May 7 - 9 2018 | San Diego Supercomputer Center, UCSD | San Diego, CA}
\begin{tightemize}
\item Learned applications of the PDB's MMTF format for structural data mining.
\item Learned to use Spark with Python for big data analysis. \item Hackathon project:  script to characterize close contacts in the PDB. \end{tightemize}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Presentations
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Posters \& Presentations} 
\begin{tabular}{rll}
March 2018      & OpenEye CUP XVIII | Santa Fe, NM \\
                & Automated workflow for reproducible analysis of protein-ligand \\
                & scoring functions.\\
March 2017      & 12th Annual NMBIST Symposium on Microbiomics | Santa Fe, NM \\
                & In silico evaluation of human OX2 orexin receptor (hOX2R) \\
                & inhibitors using AutoDock and Glide. \\
%November 2016   & 19th UTEP/NMSU Workshop on Mathematics, Computer Science, \\
%                & and Computational Sciences | El Paso, TX \\
%                & Comparison of AutoDock and Glide in docking Suvorexant to the \\
%                & crystal structure of human OX2 receptor (hOX2R). \\
October 2015    & UTEP COURI Symposium | El Paso, TX \\
                & A point mutation in the Phi-EL chaperonin alters ring separation \\
                & and protein folding activity. \\
\end{tabular}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Organizations
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{minipage} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{minipage}[t]{0.33\textwidth} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     LINKS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Links} 
Github:// \href{https://github.com/ravila4}{\bf ravila4}\\
Twitter://  \href{https://twitter.com/ravila_4}{\bf @ravila\_4}\\
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     COURSEWORK
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Coursework}
\subsection{Graduate}
Comp. Modeling of Biostructures\\
Statistical Analysis \\
Statistical Programming \\
Bioinformatics I \& II \\
Postgenomic Analysis\\
Data Mining \\
Machine Learning \\
Database Management\\
Systems Neuroscience\\
\sectionsep

\subsection{Undergraduate}
Biochemistry I \& II \\
Structural Biochemistry\\
Molecular Cell Biology \\
Prokaryotic Molecular Biology \\
Organic Chemistry I \& II \\
Physical Chemistry II \\
% Molecular Symmetry \& Group Theory \\
Discrete Mathematics \\
Data Structures and Algorithms\\
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     SKILLS
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Skills}

\subsection{Laboratory}
 \textbullet{} protein expression
 \textbullet{} SDS-PAGE\\
 \textbullet{} UV-Vis
 \textbullet{} crystal screening
 \textbullet{} HPLC\\
% \textbullet{} PCR
% \textbullet{} plasmid purification
% \textbullet{} NMR \\
\sectionsep

\subsection{Programming}
 \textbullet{} Python
 \textbullet{} UNIX Shell
 %\textbullet{} Java
 \textbullet{} R \\
 \textbullet{} MySQL
\sectionsep

\subsection{Software \& Toolkits}
 %\textbullet{} Git
 %\textbullet{} Docker
 \textbullet{} Chimera
 \textbullet{} PyMol\\
 %\textbullet{} RDKit
 %\textbullet{} AutoDock
 %\textbullet{} OpenBabel
 %\textbullet{} Rosetta
 %\textbullet{} EMAN2
 %\textbullet{} Relion
 %\textbullet{} VMD
 \textbullet{} Schrödinger
 \textbullet{} CCG MOE
 \\
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     LANGUAGES
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Languages}
\location{Native fluency:}
 \textbullet{}   English \textbullet{} Spanish \\

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     COLUMN TWO
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\end{minipage} 
\hfill
\begin{minipage}[t]{0.66\textwidth} 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Objective
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{About Me}
Biochemist with knowledge of structural biochemistry, bioinformatics, and a mixture of wet/dry lab experience. Interested in drug discovery research and protein design. %applying data mining and machine learninghttps://www.overleaf.com/project/5aaf38b1fcb2803f241ffe81 algorithms to problems in structural bioinformatics, drug discovery, and high dimensional -omics data analysis.
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Work Experience
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Work Experience}
\runsubsection{GlaxoSmithKline, via Atrium}
\descript{| Protein Design and Informatics Intern}
\location{May 2018 – August 2018 | Upper Providence, PA}
\sectionsep
\begin{tightemize}
\item Evaluated methods for identifying cryptic binding pockets in proteins.\item Wrote scripts to automate analysis of molecular dynamics (MD) data. \item Ran scalability benchmarks to optimize MD simulation performance.  \end{tightemize}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     RESEARCH
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Research}
\runsubsection{Structure-Based Drug Discovery Lab}
\descript{| Graduate RA}
\location{May 2016 – Present | UTEP School of Pharmacy | El Paso, TX}
\begin{tightemize}
\item Development and testing of novel machine-learning scoring functions for the prediction of drug-target binding affinities. \item Learned to utilize the high performance computing resources at the Texas Advanced Computing Center (TACC). \item Developed pipeline for in silico drug discovery targeting GPCRs.% \item Contributed to open source projects.
\end{tightemize}
\sectionsep

%\runsubsection{Structure-Based Drug Discovery}
%\descript{| Self-Guided Graduate Researcher}
%\location{May 2016 – December 2017 | UTEP Bioinformatics Program | El Paso, TX}
%\begin{tightemize}
%\item  Worked on putting together a workflow for virtual screening %drugs to GPCR targets.\item Learned to use several tools for %molecular docking and virtual screening, including AutoDock and %Schrodinger's drug discovery suite. \end{tightemize}
%\sectionsep

\runsubsection{Structural Biochemistry Lab}
\descript{| Undergraduate RA}
\location{March 2014 – August 2015 | UTEP Chemistry Department | El Paso, TX}
\begin{tightemize}
\item Transformation of recombinant plasmids in BL21 cells. \item Bacterial expression and purification of chaperonin proteins.\item Functional assays to evaluate protein folding activity. \item Cryo-EM image processing for 3D structure determination.\end{tightemize}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Workshops
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Workshops}
\runsubsection{MMTF Workshop \& Hackathon}\\
\location{May 7 - 9 2018 | San Diego Supercomputer Center, UCSD | San Diego, CA}
\begin{tightemize}
\item Learned applications of the PDB's MMTF format for structural data mining.
\item Learned to use Spark with Python for big data analysis. \item Hackathon project:  script to characterize close contacts in the PDB. \end{tightemize}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Presentations
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{Posters \& Presentations} 
\begin{tabular}{rll}
March 2018      & OpenEye CUP XVIII | Santa Fe, NM \\
                & Automated workflow for reproducible analysis of protein-ligand \\
                & scoring functions.\\
March 2017      & 12th Annual NMBIST Symposium on Microbiomics | Santa Fe, NM \\
                & In silico evaluation of human OX2 orexin receptor (hOX2R) \\
                & inhibitors using AutoDock and Glide. \\
%November 2016   & 19th UTEP/NMSU Workshop on Mathematics, Computer Science, \\
%                & and Computational Sciences | El Paso, TX \\
%                & Comparison of AutoDock and Glide in docking Suvorexant to the \\
%                & crystal structure of human OX2 receptor (hOX2R). \\
October 2015    & UTEP COURI Symposium | El Paso, TX \\
                & A point mutation in the Phi-EL chaperonin alters ring separation \\
                & and protein folding activity. \\
\end{tabular}
\sectionsep

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%     Organizations
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\end{minipage} 

\end{document}